make_queue/static/make_queue/js/stream.js

```diff
@@ -1,7 +1,7 @@
 function setupSocket($elem) {
     const chatSocket = new WebSocket(
         `wss://${window.location.host}/ws/stream/`
```

Comment:
When reviewing this, a tip is to change the url manually, so that you can see the streams in action.

```suggestion
        `wss://makentnu.no/ws/stream/`
```